src/services/business.services.ts

```diff
@@ -11,12 +11,10 @@ export default class BusinessServices {
   /** Creates a temporary (draft) business record. */
   static async createBusiness (businessRequest: BusinessRequest): Promise<any> {
     return axios.post(`${BusinessServices.legalApiUrl}/businesses?draft=true`, businessRequest)
-      .catch(error => error?.response)
```

Comment:
Legal API errors don't always return a response, so let the caller handle this exception instead.